### PR TITLE
[1.4.1] Avoiding npm substitution attacks & upgrade more deps

### PR DESCRIPTION
**Pull request by JungMinu**
See documentation: https://docs.google.com/document/d/1SJJ9LfZvxXPp7wyzQaTyYQJU_8fadqHHY3oU6YRdIeo/edit?usp=sharing
ncm-install-flowchart

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "p-defer": "^1.0.0",
     "proxy-agent": "^3.1.0",
     "proxy-from-env": "^1.0.0",
+    "rc": "^1.2.8",
```
Comment (edsadr) [inline package.json:49]: Did we check the license for this dep and its tree? just asking to make sure

**Review by edsadr — COMMENTED** [review]
Just an open question, the code looks good

**Comment by edsadr** [comment]
Also let's give it a check, update and fix the CI

File: package.json (hunk 1 above)
Comment (JungMinu) [reply package.json:49]: yes, it is licensed under the two-clause BSD License, MIT License

**Comment by JungMinu** [comment]
@edsadr this PR is ready to be merged now

**Pull request merged**